### PR TITLE
fix(route): tju oaa news and notification url

### PR DESCRIPTION
**Pull request by superpung**
## 完整路由地址 / Example for the proposed route(s)

```routes
/tju/oaa/news
/tju/oaa/notification
```

## 新 RSS 检查列表 / New RSS Script Checklist
  
- [x] 新的路由 New Route
  - [x] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [x] 文档说明 Documentation
  - [x] 中文文档 CN
  - [x] 英文文档 EN
- [ ] 全文获取 fulltext
  - [ ] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [x] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [x] 可以解析 Parsed
  - [x] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

## 说明 / Note

修改了已有的天津大学教务处 RSS，原 URL 已失效。
Modified the existing RSS of the Academic Affairs Office of Tianjin University, the original URL has become invalid.

### CODE REVIEW

**Comment by SuperPung** [comment]
@TonyRL , could you tell me why the pr was closed? 🥺

**Comment by SuperPung** [comment]
> #7671
> 
> You used ```route``` , it should be ```routes```

Oh sorry, I have fixed it. Could you please reopen this pr? @TonyRL very thanks to you!

File: docs/university.md [hunk 1]
```diff
@@ -2387,7 +2387,7 @@ jsjxy.hbut.edu.cn 证书链不全，自建 RSSHub 可设置环境变量 NODE_TLS
 
 ### 天津大学教务处
 
-<Route author="AmosChenYQ" example="/tjpyu/ooa/news" path="/tjpyu/ooa/:type" :paramsDesc="['默认为`news`']">
+<Route author="AmosChenYQ SuperPung" example="/tju/oaa/news" path="/tju/oaa/:type" :paramsDesc="['默认为`news`']">
```
Comment (TonyRL) [inline docs/university.md:2390]:
```suggestion
<Route author="AmosChenYQ SuperPung" example="/tju/oaa/news" path="/tju/oaa/:type?" :paramsDesc="['默认为`news`']">
```
Path here should equal to `router.js` and `maintainer.js`

File: docs/en/university.md [hunk 2]
```diff
@@ -46,6 +46,18 @@ Note: [Source website](https://gs.bjtu.edu.cn/) only provides articles in Chines
 
 <RouteEn author="exuanbo" example="/polimi/news" path="/polimi/news/:language?" :paramsDesc="['English language code en']" />
 
+## Tianjin University
+
+### The Office of Academic Affairs
+
+<RouteEn author="AmosChenYQ SuperPung" example="/tju/oaa/news" path="/tju/oaa/:type" :paramsDesc="['default `news`']">
```
Comment (TonyRL) [inline docs/en/university.md:53]:
```suggestion
<RouteEn author="AmosChenYQ SuperPung" example="/tju/oaa/news" path="/tju/oaa/:type?" :paramsDesc="['default `news`']">
```

File: lib/v2/tju/oaa/index.js [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+            list.map((item) => {
+                switch (item.type) {
+                    case 'wechat-mp':
+                        return finishArticleItem(ctx, item);
+                    case 'tju-oaa':
+                    case 'in-site':
+                        return ctx.cache.tryGet(item.link, async () => {
+                            let detailResponse = null;
+                            try {
+                                detailResponse = await got({ url: item.link, https: { rejectUnauthorized: false } });
```
Comment (TonyRL) [inline lib/v2/tju/oaa/index.js:93]:
```suggestion
                                detailResponse = await got(item.link, { https: { rejectUnauthorized: false } });
```

File: lib/v2/tju/oaa/index.js [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+    }
+    let response = null;
+    try {
+        response = await got({
+            method: 'get',
+            url: oaa_base_url + path,
+            headers: {
+                Referer: oaa_base_url,
+            },
+        });
```
Comment (TonyRL) [inline lib/v2/tju/oaa/index.js:49]: got requests can be simplified
```suggestion
        response = await got(oaa_base_url + path, {
            headers: {
                Referer: oaa_base_url,
            },
        });
```

**Pull request merged**